package.json

```diff
@@ -29,7 +29,6 @@
     "example": "examples"
   },
   "scripts": {
-    "postinstall": "[ \"$NODE_ENV\" != production ] && npm ln ../jsdom && npm ln ../buble-tape",
```

Comment:
i'm using yarn, and this command always crashed and i deleted it rather than spend time debugging since it didn't seem necessary. are  jsdom and buble-tape specified properly?